data/sql_updates/create_committee_history.sql

```diff
@@ -21,9 +21,9 @@ with
         group by committee_id
     ),
     dcp_original as (
-        select cmte_sk, min(receipt_dt) receipt_dt from dimcmteproperties
-        where form_tp = 'F1'
-        group by cmte_sk
+        select committee_id, min(receipt_date) receipt_date from vw_filing_history
+        where report_year >= :START_YEAR
```

Comment:
Should we filter this by form type at all? If we don't filter, how about updating the documentation to indicate that `first_file_date` indicates the first form of any type filed by the committee?